docs/tutorials/load-test-grpc.md

```diff
@@ -16,44 +16,129 @@ Load test a Kubernetes gRPC service and validate its [service-level objectives (
+=== "Unary example"
+    ```shell
+    iter8 k launch \
+    --set "tasks={ready,grpc,assess}" \
+    --set ready.deploy=routeguide \
+    --set ready.service=routeguide \
+    --set ready.timeout=60s \
+    --set grpc.host="routeguide.default:50051" \
+    --set grpc.call="routeguide.RouteGuide.GetFeature" \
+    --set grpc.protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/route_guide/routeguide/route_guide.proto" \
+    --set grpc.dataURL="https://raw.githubusercontent.com/iter8-tools/docs/main/samples/grpc-payload/unary.json" \
+    --set assess.SLOs.upper.grpc/error-rate=0 \
+    --set assess.SLOs.upper.grpc/latency/mean=200 \
+    --set assess.SLOs.upper.grpc/latency/p'97\.5'=800 \
+    --set runner=job
+    ```
+
+    ??? note "About this experiment"
+        This experiment consists of three [tasks](../getting-started/concepts.md#iter8-experiment), namely, [ready](../user-guide/tasks/ready.md), [grpc](../user-guide/tasks/grpc.md), and [assess](../user-guide/tasks/assess.md). 
+        
+        The [ready](../user-guide/tasks/ready.md) task checks if the `routeguide` deployment exists and is available, and the `routeguide` service exists. 
+        
+        The [grpc](../user-guide/tasks/grpc.md) task sends call requests to the `routeguide.RouteGuide.GetFeature` method of the cluster-local gRPC service with host address `routeguide.default:50051`, and collects [Iter8's built-in gRPC load test metrics](../user-guide/tasks/grpc.md#metrics). 
+        
+        The [assess](../user-guide/tasks/assess.md) task verifies if the app satisfies the specified SLOs: i) there are no errors, ii) the mean latency of the service does not exceed 50 msec, and iii) the `97.5`th percentile latency does not exceed 200 msec. 
+        
+        This is a [single-loop](../getting-started/concepts.md#iter8-experiment) [Kubernetes experiment](../getting-started/concepts.md#kubernetes-experiments) where all the previously mentioned tasks will run once and the experiment will finish. Hence, its [runner](../getting-started/concepts.md#runners) value is set to `job`.
+
+=== "Server streaming example"
+    ```shell
+    iter8 k launch \
+    --set "tasks={ready,grpc,assess}" \
+    --set ready.deploy=routeguide \
+    --set ready.service=routeguide \
+    --set ready.timeout=60s \
+    --set grpc.host="routeguide.default:50051" \
+    --set grpc.call="routeguide.RouteGuide.ListFeatures" \
+    --set grpc.protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/route_guide/routeguide/route_guide.proto" \
+    --set grpc.dataURL="https://raw.githubusercontent.com/iter8-tools/docs/main/samples/grpc-payload/server.json" \
+    --set assess.SLOs.upper.grpc/error-rate=0 \
+    --set assess.SLOs.upper.grpc/latency/mean=200 \
+    --set assess.SLOs.upper.grpc/latency/p'97\.5'=800 \
+    --set runner=job
+    ```
+
+    ??? note "About this experiment"
```

Comment:
Let us combine the "About this experiment" notes across the 4 tabs for gRPC and 2 tabs for HTTP.